test/interceptors/cache.js

```diff
@@ -1453,7 +1456,7 @@ describe('Cache Interceptor', () => {
         res.end(body)
       }).listen(0)
 
-      const client = new Client(`http://localhost:${server.address().port}`)
+      const client = new Client(`http://localhost:${server.address().port}`, { maxRedirections: 0 })
```

Comment:
Since the test includes redirect responses (furthermore, invalid ones), explicitly disable following redirections.

Reply:
Max redirections is not supported anymore, you'd require to use the interceptor instead.

Reply:
If I understand it well, it means that redirection are not followed by default, an interceptor has to be added for that. Fine for the test then.

Reply:
I have removed this option.